app/controllers/integrations/buildkite_controller.rb

```diff
@@ -28,4 +28,9 @@ def build_param
   def message
     build_param[:message]
   end
+
+  def release_params
+    extra_params = Samson::Hooks.fire(:release_params, project, build_param)
```

Comment:
Looking back at your previous comment, I'm not sure if you mean that the method should be renamed to `buildkite_release_params`, or the hook. Could you clarify?

On the decorator idea: this sounds similar to what we were doing at ff9c5a7ef5bb54961ea47d07c87a283ba176b7fb, where you suggested using a hook. We liked that idea; by using a hook, we were hoping to introduce a more general way to pull in the build number, as opposed to something specific to Buildkite. Has your opinion changed?

It might be useful for you to see the plugin we're working on, for some context: https://github.com/redbubble/samson-release-number-from-ci uses the hook, and the previous version https://github.com/redbubble/samson-release-number-from-ci/tree/150797f1588faae0537dd24d4ba538e8bc263ccc uses a decorator.